configure_ironic.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
```

Comment:
We have 2 of these files in this patch
./configure_ironic.sh and ./config/configure_ironic.sh

Reply:
aha!  sorry about that.  I think we should land the 'split containers' patch first and then I'll rework this.